src/views/catalog/CreateFromInstanceTypes/hooks/useEnableInstanceTypeModal.ts

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from 'react';
+
+import { INSTANCE_TYPE_ENABLED } from '@kubevirt-utils/hooks/useFeatures/constants';
+import { useFeatures } from '@kubevirt-utils/hooks/useFeatures/useFeatures';
+
+export type EnableInstanceTypeTechPreviewModalProps = {
+  canEdit: boolean;
+  isOpen: boolean;
+  onClose: () => void;
+  onEnableInstanceTypeFeature: () => void;
+};
+
+type UseEnableInstanceTypeModal = (
+  isInstanceTypeTab: boolean,
+  navigateToCatalog: () => void,
+) => EnableInstanceTypeTechPreviewModalProps & { loading: boolean };
```

Comment:
loading here? why not line 6?

Reply:
since I wanted to re-use the Props from above, but since I'm changing, I'll change that too :)